entity-framework/core/modeling/value-comparers.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Value Comparers - EF Core
+author: ajcvickers
+ms.date: 03/17/2020
+uid: core/modeling/value-comparers
+---
+
+# Value Comparers
+
+> [!NOTE]  
+> This feature is new in EF Core 3.0.
+
+> [!TIP]  
+> The code in this document can be found on GitHub as a [runnable sample](https://github.com/dotnet/EntityFramework.Docs/tree/master/samples/core/Modeling/ValueConversions/).
+
+## Background
+
+EF Core needs to compare property values when:
+
+* Determining whether or not a property has been changed as part of [detecting changes for updates](xref:core/saving/basic)
+* Determining whether or not two key values are the same when resolving relationships 
```

Comment:
https://afterdeadline.blogs.nytimes.com/2010/03/01/whether-or-not/

Reply:
Very interesting. Fixed.